src-v2/NTS.Domain.Setup/Entities/Loop.cs

```diff
@@ -1,51 +1,58 @@
-﻿using NTS.Domain.Setup.Import;
-using Newtonsoft.Json;
+﻿using Newtonsoft.Json;
 
 namespace NTS.Domain.Setup.Entities;
-
-public class Loop : DomainEntity, ISummarizable
+public class Loop : DomainEntity, IParent<Lap>
 {
-    public static Loop Create(double distance, int recovery, int rest) => new(distance, recovery, rest);
-
-    public static Loop Update(int id, double distance, int recovery, int rest) => new(id, distance, recovery, rest);
+    public static Loop Create(double distance) => new(distance);
+    public static Loop Update(int id, double distance) => new(id, distance);
 
+    public Loop()
+    {
+        Laps = new List<Lap> { };
+    }
     [JsonConstructor]
-    public Loop(int id, double distance, int recovery, int rest) : this(distance, recovery, rest)
+    public Loop(List<Lap> laps)
```

Comment:
Wait, Loop consists of Laps? Now I am entirely confused. I don't think we need all 3 of `Phase`, `Lap` and `Loop`.